doc/release-notes/rl-2105.adoc

```diff
@@ -111,6 +111,10 @@ configure.packages.*.start  -> programs.neovim.plugins = [ { plugin = ...; }]
 configure.customRC -> programs.neovim.extraConfig
 ----
 
+* There is a new <<opt-qt.style>>, allowing the style of Qt5 applications to be set. If you use
+<<opt-qt.platformTheme>> set to `gnome` though, now it is necessary to <<opt-qt.style>> to either
+`adwaita` or `adwaita-dark`, to show the correct theme.
+
```

Comment:
Would you mind explaining where the limitation comes from :slightly_smiling_face: ?

Reply:
I am just replicating from NixOS module, but AFAIK you need to have a valid theme that matches the Gnome one and the only two that exists are `adwaita` and `adwaita-dark`.

If you want to use a custom theme you wouldn't use `platformTheme` set to `gnome`.

Reply:
Seems like `breeze` is also a valid choice: https://wiki.archlinux.org/index.php/Uniform_look_for_Qt_and_GTK_applications#Styles_for_both_Qt_and_GTK

Reply:
So I think I will change the assertion that if `qt.platformTheme` is set to "gnome", `qt.style` should be set to something.